apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/utils/UserUtils.java

```diff
@@ -0,0 +1,27 @@
+package it.pagopa.selfcare.onboarding.utils;
+
+import it.pagopa.selfcare.onboarding.common.PartyRole;
+import it.pagopa.selfcare.onboarding.config.MailTemplatePathConfig;
+import it.pagopa.selfcare.onboarding.entity.Onboarding;
+import it.pagopa.selfcare.onboarding.entity.User;
+
+import java.util.Objects;
+
+public class UserUtils {
+
+  public UserUtils() {}
+
+  public static String getEmailRegistrationTemplatePath(MailTemplatePathConfig config, Onboarding onboarding) {
```

Comment:
you could Create unit tests for UserUtils.getEmailRegistrationTemplatePath to verify its behavior under various scenarios, including edge cases....in addition, you could add some comments to the UserUtils method to explain its purpose, parameters, and return values for future maintainers.